src/sys/windows/net.rs

```diff
@@ -3,12 +3,16 @@ use std::mem;
 use std::net::SocketAddr;
 use std::sync::Once;
 
-use winapi::ctypes::c_int;
-use winapi::shared::in6addr::{in6_addr_u, IN6_ADDR};
-use winapi::shared::inaddr::{in_addr_S_un, IN_ADDR};
-use winapi::shared::ws2def::{ADDRESS_FAMILY, AF_INET, AF_INET6, SOCKADDR, SOCKADDR_IN};
-use winapi::shared::ws2ipdef::{SOCKADDR_IN6_LH_u, SOCKADDR_IN6_LH};
-use winapi::um::winsock2::{ioctlsocket, socket, FIONBIO, INVALID_SOCKET, SOCKET};
+use windows_sys::Win32::Networking::WinSock::{
+    ioctlsocket, socket, FIONBIO, IN6_ADDR, IN6_ADDR_0, INVALID_SOCKET, IN_ADDR, IN_ADDR_0,
+    SOCKADDR, SOCKADDR_IN, SOCKADDR_IN6, SOCKADDR_IN6_0, SOCKET,
+};
+
+// AF_INET/6 are in the Win32_NetworkManagement_IpHelper feature for some reason,
+// so just directly define them here rather than import that whole feature set
+// these constants won't ever change
```

Comment:
Although I get not wanting to add a feature for two constant I don't want to keep these constant in Mio, that should be done in `windows-sys`.

Reply:
Feel free to suggest a more appropriate feature here: https://github.com/microsoft/win32metadata